### PR TITLE
docs: remove @angular/platform-webworker from comment

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/common/src/location/platform_location.ts [hunk 1]
```diff
@@ -18,8 +18,7 @@ import {DOCUMENT} from '../dom_tokens';
  * agnostic.
  * This means that we can have different implementation of `PlatformLocation` for the different
  * platforms that angular supports. For example, `@angular/platform-browser` provides an
- * implementation specific to the browser environment, while `@angular/platform-webworker` provides
- * one suitable for use with web workers.
+ * implementation specific to the browser environment
```
Comment (kapunahelewong) [inline packages/common/src/location/platform_location.ts]:
```suggestion
 * implementation specific to the browser environment.
```

**Review by kapunahelewong — APPROVED** [review]
Thank you, @santoshyadav198613. Just a small fix. We'll still need engineering to review.

File: packages/common/src/location/platform_location.ts (hunk 1 above)
Comment (santoshyadav198613) [reply packages/common/src/location/platform_location.ts]: Done.

**Review by kara — CHANGES_REQUESTED** [review]
@santoshyadav198613 Can you update the commit message with more information on why we should remove the mention of platform-webworker?

**Comment by santoshyadav198613** [comment]
> @santoshyadav198613 Can you update the commit message with more information on why we should remove the mention of platform-webworker?

Done.

**Comment by santoshyadav198613** [comment]
Hi @kara,
Can you check now, I have updated the message.

**Comment by santoshyadav198613** [comment]
Hi @kapunahelewong ,
Can we get it reviewed, let me know if any changes are required.

**Comment by santoshyadav198613** [comment]
Hi @kara ,
Can you please review and check if anything else is needed.

**Comment by santoshyadav198613** [comment]
Hi Team,
Can anyone look into this PR please.

**Comment by santoshyadav198613** [comment]
Closing this PR as it was already removed.

**Pull request closed without merging**